src/pages/pool/withdraw.vue

```diff
@@ -24,6 +25,7 @@ const { network } = configService;
 const { pool, poolQuery, loadingPool, transfersAllowed } = usePoolTransfers();
 const { isDeepPool } = usePool(pool);
 const { activeTab, resetTabs } = useWithdrawPageTabs();
+usePoolTransfersGuard();
```

Comment:
I don't think we should ever guard against withdrawals. If a user has funds in a pool they should always be able to access this page. Do you know if we did this before?

Reply:
Previously it was in the `src/pages/_layouts/PoolTransferLayout.vue` component, so it affected both join and exit pages. But I think it's not needed for exit page

Reply:
Yeah see that now, it's fine to leave it in.